common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -44,3 +50,103 @@ innovation_war_camels = {
 	flag = global_regional
 	flag = tribal_era_regional
 }
+###Tech
+##Tribal
+innovation_kyparite_sap = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = custom_kyparite_regions
+	icon = @kyparite_sap
+
+	modifier = {
+		garrison_size = 0.02
+		skirmishers_damage_mult = 0.15
+		skirmishers_toughness_mult = 0.15
+		monthly_prestige_gain_mult = 0.02
+		monthly_piety_gain_mult = 0.02
+		
+	}
+
+	flag = global_regional
+	flag = tribal_era_regional
+}
+innovation_sea_faring = {
```

Comment:
Exist in 2 files.
The tech from this file has different bonuses than the other tech.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/110716315-587bba00-8220-11eb-86c6-79474bb74471.png)

</details>